facebook/delphiFacebook/R/contingency_write.R

```diff
@@ -109,15 +107,62 @@ add_geo_vars <- function(data, params, geo_type) {
         .data$county_fips
       )
   } else if (geo_type == "county") {
-    warning("county metadata not supported")
+    counties <- read_csv(
+      file.path(params$static_dir, "02_20_uszips.csv"),
+      col_types = cols(.default = "c")
+    ) %>% 
+      mutate(
+        fips = stri_pad(.data$fips, 5, pad="0")
+      ) %>%
+      select(fips, county_name, state_id, state_name) %>% 
+      distinct()
+
+    rest <- data.frame(
+      county_fips = data$geo_id
+    )
+
+    rest <- left_join(rest, counties, by = c("county_fips" = "fips")) %>%
+      mutate(
+        region = .data$state_id,
+        state = .data$state_id,
+        county = .data$county_name,
+        county_fips = .data$county_fips
+      ) %>% 
+      select(
+        .data$region,
+        .data$state,
+        .data$county,
+        .data$county_fips
+      )
```

Comment:
`select` can rename columns while selecting (e.g. `select(foo = bar, baz = spam)`), so is it necessary to have two steps here?

Reply:
Ah, didn't know that, I'll combine the steps.